### PR TITLE
Alerts and buttons

### PR DESCRIPTION
**Pull request by irajbhinde**
Hey! Please review my code & let me know if any changes are to be made 

Added Components - 

**Alerts**

1. Simple Alert
2. Outline Alert

**Buttons**

1. Primary & Secondary Button
2. Link Button
3. Floating Action Button 

**GIFY** - 

alerts_and_buttons

### CODE REVIEW

File: Components/Alerts/alerts.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+.header_content.warning{
+    background-color: #ff9800;
+    color: rgb(247, 234, 234);
+}
+
+.header_content.outline-danger{
+  color: #f44336;
+  border: 1px solid #f44336 ;
```
Comment (syed-shaida-hussain) [inline Components/Alerts/alerts.css]: try to use variables for colors everywhere.

File: Components/Buttons/buttons.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+.icon-btn{
+    background-color: var(--primary-color);
+    color: white;
+}
+
+.link{
+    background-color: #6F99DE;
+    color: white;
+
```
Comment (syed-shaida-hussain) [inline Components/Buttons/buttons.css]: use variables for colors here as well.

**Review by syed-shaida-hussain — CHANGES_REQUESTED** [review]
Try to use CSS variables for colors everywhere in your code as it would make it more generic.

File: Components/Alerts/alerts.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
+.header_component {
+
```
Comment (shrushti2000) [inline Components/Alerts/alerts.css]: remove extra / unused lines

File: Components/Alerts/alerts.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+
+/*   
+  
+  .main_content {
+    width: 500px;
+    height: 200px;
+    overflow-y: scroll;
+  } */
```
Comment (shrushti2000) [inline Components/Alerts/alerts.css]: remove the commented code part.

File: Components/Buttons/buttons.html [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+            <button class="btn floatin-compass"><i class="fas fa-compass fa-2x"></i></button>
+        </div>
+
+
+
+
+
+      </div>
+      </div>
+
```
Comment (shrushti2000) [inline Components/Buttons/buttons.html]: remove the extra lines.

File: Components/Buttons/buttons.css [hunk 6]
```diff
@@ -0,0 +1,54 @@
+.btn{
+    
+    min-width: 100px;
+    width: max-content;
+    padding: 0.8rem;
+    border-radius: 2px;
+    border: none;
+}
```
Comment (shrushti2000) [inline Components/Buttons/buttons.css]: try using relative units and again remove the  extra line here

File: Components/Buttons/buttons.css [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+
+.primary-btn{
+    background-color: var(--primary-color);
+    color: white;
+}
+
+.secondary-btn{
+    background-color: #EBBA57 ;
+    color: white;
```
Comment (shrushti2000) [inline Components/Buttons/buttons.css]: try using CSS variables for colors .

**Review by shrushti2000 — COMMENTED** [review]
Suggested few minor changes relating to code  formatting. Overall its good. Great work.

**Pull request merged**